inst/qml/common/GlmInputComponent.qml

```diff
@@ -0,0 +1,210 @@
+//
+// Copyright (C) 2013-2018 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+
+import QtQuick			2.8
+import QtQuick.Layouts  1.3
+import JASP.Controls	1.0
+import JASP.Widgets		1.0
+import JASP				1.0
+
+// All Analysis forms must be built with the From QML item
+Group {
+
+	function updateWeightsTitle() {
+		if (family.currentText == "Binomial")
```

Comment:
```suggestion
		if (family.currentText === "Binomial")
```
I think you can use `===` instead of `==` almost everywhere.

Reply:
Does it matter whether === or == is used in this case?

Reply:
well, it upsets the default Qt linter:
![image](https://user-images.githubusercontent.com/21319932/167397130-19461d87-59fe-4f5d-9ef9-a0d83f972cc3.png)

So Qt recommends to not write QML like that. There may also be performance benefits of `===` over `==`, but that's pure speculation and depends on the QML compiler. In this case, there is probably no practical difference, otherwise, you would have noticed this in jasp.

Reply:
Ah thanks for the explanation. I changed it. :)